benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -623,6 +612,10 @@ void GraphicsBenchmarkApp::SetupSpheresPipelines()
     piCreateInfo.setCount                          = 1;
     piCreateInfo.sets[0].set                       = 0;
     piCreateInfo.sets[0].pLayout                   = mSphere.descriptorSetLayout;
+    piCreateInfo.pushConstants.count               = sizeof(float3) / sizeof(uint32_t);
```

Comment:
with the padding, you can change this to sizeof(float4)/sizeof(uint32_t)

Reply:
maybe better to add a const like kDebugColorPushConstantCount = sizeof(float4)/sizeof(uint32_t) so that you can also use this kDebugColorPushConstantCount later when PushGraphicsConstants is called

Reply:
Good idea. Done.